source/adapters/level_zero/command_buffer.cpp

```diff
@@ -407,7 +409,7 @@ urCommandBufferCreateExp(ur_context_handle_t Context, ur_device_handle_t Device,
   // Create signal & wait events to be used in the command-list for sync
   // on command-buffer enqueue.
   auto RetCommandBuffer = *CommandBuffer;
-  UR_CALL(EventCreate(Context, nullptr, true, &RetCommandBuffer->SignalEvent));
+  UR_CALL(EventCreate(Context, nullptr, false, &RetCommandBuffer->SignalEvent));
```

Comment:
Are you sure there is not use cases where the host needs to access the SignalEvent? E.g. post-process the data on the host?

Reply:
This event is never returned by to the user, so it would only be our runtime code that would want to access `SignalEvent`. We currently don't have any calls to `zeEventHostSignal` or `zeEventHostSynchronize` in our code to need host visibility, but if we wanted to use these for post-processing, then we can set this back to `true` as part of that future PR.